src/gantry/screens.py

```
@@ -666,6 +675,125 @@ def action_focus_search(self) -> None:
         search_input.focus()
         self.search_active = True
 
+    def action_focus_filter(self) -> None:
+        """Show and focus the advanced filter panel."""
+        filter_panel: FilterPanel = self.query_one("#filter-panel", FilterPanel)
+        filter_panel.add_class("show")
+        try:
+            inp = filter_panel.query_one("#filter-expr-input")
+            inp.focus()
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)